solidity/package.json

```diff
@@ -8,7 +8,7 @@
     "test": "npx hardhat test --network localhost",
     "typechain": "npx hardhat typechain",
     "compile-deployer": "npx tsc -p . ; pkg dist/contract-deployer.js --targets node10-linux-x64",
-    "evm_fork": "npx hardhat node --fork https://eth-mainnet.alchemyapi.io/v2/${ALCHEMY_ID} --fork-block-number 11780000",
+    "evm_fork": "npx hardhat node --fork https://eth-mainnet.alchemyapi.io/v2/${ALCHEMY_ID} --fork-block-number 12782071",
```

Comment:
notice the solidity arbitrary logic test is failing, I suggest we change it's script to use solidity_test_fork and keep evm fork for our new test runner test. That way we can use the pinned block that we updated here in that old test and keep it working.